ParquetClassLibrary/Characters/Character.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox;
+using ParquetClassLibrary.Sandbox.IDs;
+using ParquetClassLibrary.Utilities;
+
+namespace ParquetClassLibrary.Characters
+{
+    public abstract class Character : Being
+    {
+        #region Class Defaults
+        /// <summary>A pronoun to use when none is specified.</summary>
+        // TODO This is just a place-holder, I am not sure yet how we will handle pronouns.
+        public const string DefaultPronoun = "they";
+        #endregion
+
+
+        #region Characteristics
+        /// <summary>The pronouns the <see cref="Character"/> uses.</summary>
+        // TODO This is just a place-holder, I am not sure yet how we will handle pronouns.
+        public string Pronoun { get; }
+
+        /// <summary>Types of parquets this <see cref="Character"/> avoids.</summary>
+        public readonly List<EntityID> Quests = new List<EntityID>();
+
+        /// <summary>Dialogue lines this <see cref="Character"/> can say.</summary>
+        // TODO This is just a place-holder, I am not at all sure how we will handle this.
+        public readonly List<string> Dialogue = new List<string>();
+
+        /// <summary>This <see cref="Character"/>'s belongings.</summary>
+        // TODO This is just a place-holder, inventory may need its own class.
+        public readonly List<EntityID> Inventory = new List<EntityID>();
+        #endregion
+
+        #region Initialization
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Character"/> class.
+        /// </summary>
+        /// <param name="in_bounds">
+        /// The bounds within which the <see cref="Character"/>'s <see cref="EntityID"/> is defined.
+        /// Must be one of <see cref="AssemblyInfo.BeingIDs"/>.
+        /// </param>
+        /// <param name="in_id">Unique identifier for the <see cref="Character"/>.  Cannot be null.</param>
+        /// <param name="in_name">Player-friendly name of the <see cref="Character"/>.  Cannot be null or empty.</param>
+        /// <param name="in_nativeBiome">The <see cref="Biome"/> in which this <see cref="Character"/> is most comfortable.</param>
```

Comment:
Can someone only appreciate one Biome, or is a List more descriptive? Right now, Biomes are fairly dramatically different, so it'd be reasonable if it was confined to one Biome per creature.

This comment applies to Critters too, now that I think on it.

Reply:
HMMMMM.  I feel like just one makes the most sense but it would be way easier to update this now than later....

Thanks, good point, I'll consider it!